src/test/scala/io/opentargets/openfda/utils/MathUtilsTest.scala

```diff
@@ -0,0 +1,39 @@
+package io.opentargets.openfda.utils
+
+import breeze.linalg.{DenseMatrix => BDM, DenseVector => BDV}
+import org.scalatest.flatspec.AnyFlatSpecLike
+import org.scalatest.matchers.should.Matchers
+
+import scala.util.Random
+
+class MathUtilsTest extends AnyFlatSpecLike with Matchers {
+  import breeze.linalg._
+
+  "Each row of rmultinomial probabilities" should "sum to 'size' where 'size' is the number of options" in {
+    val size = 10
+    val multis: BDM[Double] =
+      MathUtils.rmultinom(10, size, BDV(Seq.fill(10)(Random.nextDouble).toArray))
```

Comment:
not sure what you want to achieve here is this? `BDV.rand[Double](10)` but the function is waiting for a double already so it will be inferred so `BDV.rand(10)`. This is a good sheet about [Breeze about linear algebra and miscellanea ](https://github.com/scalanlp/breeze/wiki/Breeze-Linear-Algebra)

Reply:
Unfamiliarity with the Breeze library: fixed now.